app/src/main/java/com/teamltt/carcare/fragment/RealTimeUpdates.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2017, Team LTT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.teamltt.carcare.fragment;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.os.Handler;
+import android.provider.ContactsContract;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.jjoe64.graphview.GraphView;
+import com.jjoe64.graphview.series.DataPoint;
+import com.jjoe64.graphview.series.LineGraphSeries;
+import com.teamltt.carcare.R;
+import com.teamltt.carcare.activity.HomeActivity;
+
+import java.util.Random;
+
+public class RealTimeUpdates extends Fragment {
```

Comment:
I believe this file was used by @anoll3 when we first were getting home screen graphs to work. It would be replaced by MyGraphAdapter's ViewHolder inner class.